pocs/camera/libasi.py

```diff
@@ -40,35 +39,47 @@ def __init__(self, library_path=None, *args, **kwargs):
                 locate the library.
             OSError: raises if the ctypes.CDLL loader cannot load the library.
         """
-        super().__init__(*args, **kwargs)
-        library = load_library(name='ASICamera2', path=library_path, logger=self.logger)
-        self._CDLL = library
-        self._version = self.get_SDK_version()
-
-    # Properties
-
-    @property
-    def version(self):
-        """ Version of the ZWO ASI SDK """
-        return self._version
+        super().__init__(name='ASICamera2', path=library_path, **kwargs)
```

Comment:
No idea. The library that ships with the current version of the ZWO SKD is called things like `libASICamera2.so` or `libASICamera2.dylib` so I use the name `ASICamera2` here so that the `ctypes.util.find_library()` function can find it for me.